operators/src/processing/raster_vector_join/mod.rs

```diff
@@ -28,19 +29,37 @@ const MAX_NUMBER_OF_RASTER_INPUTS: usize = 8;
 
 /// The parameter spec for `RasterVectorJoin`
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
 pub struct RasterVectorJoinParams {
     /// Each name reflects the output column of the join result.
     /// For each raster input, one name must be defined.
     pub names: Vec<String>,
 
-    /// Specifies which method is used for aggregating values
-    pub aggregation: AggregationMethod,
+    /// Specifies which method is used for aggregating values for a feature
+    pub feature_aggregation: FeatureAggregationMethod,
+
+    /// Specifies which method is used for aggregating values over time
+    pub temporal_aggregation: TemporalAggregationMethod,
+}
+
+/// How to aggreagate the values for the geometries inside a feature
+/// e.g. add all the raster values corresponding to the individual
+/// points inside a `MultiPoint` feature. Choosing None leads to an
```

Comment:
i think we have to separate multi-features and things like polygons where each single feature can contain multiple pixels.
for points we will hit  only one pixel, however, lines and polygons can hit multiple pixels.